src/main/java/core/basesyntax/service/InventoryFromCsv.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.StockDao;
+import core.basesyntax.db.StockDaoStorageImpl;
+import core.basesyntax.service.operations.Balance;
+import core.basesyntax.service.operations.Operation;
+import core.basesyntax.service.operations.Purchase;
+import core.basesyntax.service.operations.Return;
+import core.basesyntax.service.operations.Supply;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class InventoryFromCsv implements Inventory {
+    private static final String FILE_NAME = "inventory.csv";

```

Comment:
Avoid hardcoding the file name directly in the class. Consider passing the file name as a parameter to the constructor or using a configuration file to manage such constants, as suggested in the checklist.